runtime/memory.c

```diff
@@ -729,6 +729,18 @@ CAMLexport caml_stat_string caml_stat_strdup(const char *s)
   return result;
 }
 
+CAMLexport caml_stat_string caml_stat_memcpy(const char *s, asize_t size,
+                                             asize_t *out_size)
+{
+  caml_stat_block result = caml_stat_alloc_noexc(size);
+  if (result == NULL)
+    caml_raise_out_of_memory();
```

Comment:
Would calling `caml_stat_alloc` directly work here?

Reply:
The `size == 0` case bothers me with that, but actually the `memcpy` change in the name still isn't correct. Honestly, what's wrong with `caml_runtime_function_0`, `caml_runtime_function_1`, ...??

Reply:
C++ name mangling schemes are even more readable!

Reply:
Changed my mind by adding an assertion on `size`! `caml_stat_memdup` captures slightly less badly the idea that that function allocates.